installation/resources/installer-cr-cluster.yaml.tpl

```diff
@@ -34,6 +34,11 @@ spec:
    #   namespace: "knative-build"
     - name: "knative-eventing"
       namespace: "knative-eventing"
+   # - name: "knative-eventing-channel-kafka"
```

Comment:
why this component is commented out?

Reply:
it was decided that this component will be not installed by default.

Reply:
The `knative-eventing-channel-kafka` component is not batteries included (we have Natss for that purpose). Also it requires user configuration. Therefore it is an optional component.